### PR TITLE
TW-410 Removed "related" from subtitle in sidebar

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/github.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ contextual_links:
   - type: section
     name: "Additional Resources"
   - type: subtitle
-    name: "Related Blog posts"
+    name: "Blog posts"
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/github.md:16]: To match capitalization on other entries

```suggestion
    name: "Blog Posts"
```

File: src/pages/docs/integrations/available-integrations/microsoft-power-automate.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ contextual_links:
   - type: section
     name: "Additional Resources"
   - type: subtitle
-    name: "Related blog posts"
+    name: "blog posts"
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/microsoft-power-automate.md:16]:
```suggestion
    name: "Blog Posts"
```

**Review by HeatherStenson — APPROVED** [review]
Lgtm! 🍊

**Review by jkonrath-postman — APPROVED** [review]
LGTM! Make sure to pull develop before you merge, because there's been a lot of drift in the last few days.

**Review by tristandenyer — APPROVED** [review]

**Pull request merged**